scipy/optimize/tnc.py

```diff
@@ -311,9 +317,12 @@ def _minimize_tnc(fun, x0, args=(), jac=None, bounds=None,
         iteration. If maxCGit == 0, the direction chosen is
         -gradient if maxCGit < 0, maxCGit is set to
         max(1,min(50,n/2)). Defaults to -1.
-    maxiter : int
-        Maximum number of function evaluation. If None, `maxiter` is
+    maxfun : int
```

Comment:
move `maxfun` to the end of the docstring, as the keyword is going to the end of the argument list.

Reply:
I moved it.

Reply:
You moved maxiter, not maxfun

Reply:
Oh, sorry. I fixed.